examples/client/spinner.cpp

```diff
@@ -26,7 +26,7 @@ static void shutdown(int)
 
 int main()
 {
-    struct sigaction action;
+    struct sigaction action = {};
```

Comment:
I think I would mildly prefer
```suggestion
    struct sigaction action{};
```
but not blocking.